controllers/main_test.go

```diff
@@ -7,8 +7,12 @@ import (
 	"github.com/aws/eks-anywhere/internal/test/envtest"
 )
 
+const integrationTestEnvVar = "INTEGRATION_TESTS_ENABLED"
+
 var env *envtest.Environment
 
 func TestMain(m *testing.M) {
-	os.Exit(envtest.RunWithEnvironment(m, envtest.WithAssignment(&env)))
+	if os.Getenv(integrationTestEnvVar) == "true" {
+		os.Exit(envtest.RunWithEnvironment(m, envtest.WithAssignment(&env)))
+	}
```

Comment:
Doesn't this skip all tests for the package? both the ones that use envtest and the ones that don't.

I could be wrong but I thought if you include a `TestMain` that executed always. And if you don't run `m`, then nothing runs.

Reply:
I think you're right, good catch. i've updated this to also be able to run without envtest.